tests/run_gen.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+from clvm_rs import run_generator, STRICT_MODE
+from clvm.operators import OP_REWRITE
+from clvm import KEYWORD_FROM_ATOM, KEYWORD_TO_ATOM
+from time import time
+from clvm_tools import binutils
+import sys
+from run import native_opcode_names_by_opcode
+
+def run_gen(fn):
+
+    # the generator ROM from:
+    # https://github.com/Chia-Network/chia-blockchain/blob/main/chia/wallet/puzzles/rom_bootstrap_generator.clvm.hex
+    program_data = bytes.fromhex(
+        "ff02ffff01ff02ff0cffff04ff02ffff04ffff02ff05ffff04ff08ffff04ff13" \
```

Comment:
FYI, python doesn't actually require `\` here because it's in parens `(...)`.

Reply:
heh, I've always been confused when exactly I need these.

Reply:
removed in https://github.com/Chia-Network/clvm_rs/pull/95/commits/3cfd2f9498648702c7bc1680ff153da60c9d15fe